clients/csrf/csrf_token_manager.go

```diff
@@ -0,0 +1,11 @@
+package csrf
+
+import "net/http"
+
+type CsrfTokenManager interface {
```

Comment:
*Manager as a suffix is not better for naming of classes or interfaces. This is the same as God. Try to rename it if the interface is referring to something more and you need to add "and" in the name, then separate the interface into two :)

Reply:
Renamed to -> Updater, that's the main functionality of this object.